src/seedu/addressbook/AddressBook.java

```diff
@@ -1072,7 +1161,16 @@ private static boolean isPersonEmailValid(String email) {
         return email.matches("\\S+@\\S+\\.\\S+"); // email is [non-whitespace]@[non-whitespace].[non-whitespace]
         //TODO: implement a more permissive validation
     }
-
+    
+    /**
+     * Returns true if the given string is a legal person birth date
+     * 
+     * @param birth date to be validated
+     * @return whether arg is a valid person birth date
+     */
+    private static boolean isPersonBirthdateValid(String birthdate) {
```

Comment:
Good job adding header comments for your function =).
The comments can be improved further by adding in what it means to be a `legal person birth date`.

Returns true if the given string is a legal person birth date.
A birth date is legal if it follows [date]/[month]/[year].